src/Microsoft.DotNet.Darc/Darc/Operations/GatherDropOperation.cs

```diff
@@ -630,7 +630,7 @@ private IEnumerable<Build> FilterReleasedBuilds(IEnumerable<Build> builds)
             {
                 if (build.Released)
                 {
-                    Console.WriteLine($"  Skipping download of released build {build.AzureDevOpsBuildNumber} of {build.GitHubRepository ?? build.AzureDevOpsRepository} @ {build.Commit}");
+                    Console.WriteLine($"  Skipping download of released build {build.AzureDevOpsBuildNumber} of {build.Repository} @ {build.Commit}");
```

Comment:
I some places we use Logger and in some places Console to write down warnings/information (esp. in the darc gather-drop command). At one point, I think, we need to review/standardize the way we log things.

Reply:
Agree, but that will need to be done carefully as some things are written directly to console because they are parsed later from the output (like JSONs or subscription information)